libraries/crypto/src/ecdh.rs

```diff
@@ -20,6 +20,7 @@ use super::rng256::Rng256;
 
 pub const NBYTES: usize = int256::NBYTES;
 
+#[derive(Clone)]
```

Comment:
It is actually only convenience for tests. We don't do conditional derive anymore, that's way it's not obvious from the code. Used here:

```rust

    #[cfg(test)]
    pub fn new_test(
        key_agreement_key: crypto::ecdh::SecKey,
        pin_uv_auth_token: [u8; PIN_TOKEN_LENGTH],
    ) -> ClientPin {
        ClientPin {
            pin_protocol_v1: PinProtocol::new_test(key_agreement_key.clone(), pin_uv_auth_token),
            pin_protocol_v2: PinProtocol::new_test(key_agreement_key, pin_uv_auth_token),
            ...
```


I could
- pass in 2 different keys
- ignore since tests run in software
? It doesn't matter anywhere if these are identical or not. Maybe for some later tests, I'll want different secret so I can check that you can't cross-combine protocols. So I'll just go with the former and remove the `Clone`?

Reply:
Clone removed, it's also a bit stricter now, and prepared for more tests.